src/config.js

```diff
@@ -1,2 +1,3 @@
 export const WIRE_TIMEOUT = 1000
+export const INSTRUMENT_TIMEOUT = 1000
```

Comment:
Merge both timeout to `ANIMATION_TIMEOUT`.
Or rename `INSTRUMENT_TIMEOUT` to ANIMATION_TIMEOUT, and let `WIRE_TIMEOUT`